.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: false
 
 language: node_js
 node_js:
-  - 13
+  - 12
```

Comment:
Temporary fix for issue with babel https://github.com/storybookjs/storybook/issues/10477